frontend-react/src/pages/deliveries/Table/DeliveriesTable.tsx

```diff
@@ -84,6 +85,9 @@ const DeliveriesTableContent: React.FC<DeliveriesTableContentProps> = ({
     const transformDate = (s: string) => {
         return new Date(s).toLocaleString();
     };
+    const handleExpirationDate = (expiresDate: string) => {
```

Comment:
Just curious, why are we putting this functionality here as opposed to within `ColumnData.tsx` directly?

Reply:
This will allow for components to handle whether to show the action button based on a function. In this case, I need to check for expiration date but other tables may want to show/hide based off another parameter.

Reply:
I think instead of passing down a chain of optional functions, and then checking if they're `undefined` or not, it would be more semantic just to have an explicit feature like `checkForExpiry` that would then implement all that logic if it's `true`.